src/main/resources/templates/db_scripts/schema.json

```diff
@@ -608,6 +620,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": false
+        },
+        {
```

Comment:
Does the `fromModuleVersion` need updating for indexes, or is there creation automatic irrespective of the `fromModuleVersion`?

Reply:
I think @julianladisch mentioned that indexes will be created always regardless of `fromModuleVersion`.

Reply:
Yes, we have automatic index recreation. For indexes `fromModuleVersion` is no longer needed.